docs/content/preview/admin/yb-admin.md

```diff
@@ -640,6 +640,36 @@ Flushed [yugabyte.table_name] tables.
 ```
 ---
 
+
+#### backfill_indexes_for_table
+
+Backfill all DEFERRED indexes in a YCQL table.
+
+**Syntax**
+
+```sh
+yb-admin \
+    -master_addresses <master-addresses> \
+    backfill_indexes_for_table <keyspace> <table_name>
+```
+
+* *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
+* *keyspace*: Specifies the keyspace `ycql.keyspace-name`.
+* *table_name*: Specifies the table name.
+
+**Example**
+
+```sh
+./bin/yb-admin \
+    -master_addresses ip1:7100,ip2:7100,ip3:7100 \
+    backfill_indexes_for_table ybdemo table_name
+```
+
+```output
```

Comment:
```suggestion
```

Reply:
you can delete the blank output box as you are already mentioning about it in the next line.

Reply:
Added in https://github.com/yugabyte/yugabyte-db/pull/20523/commits/3f018544dc8ec5d1d550e9bfdbadb00a87f8702d